docs/dashboard/configuration/shipping.md

```diff
@@ -53,17 +53,17 @@ These two methods cover most of the basic use cases, for example:
 * Increasing the shipping price for heavy orders.
 
 
-### ZIP Codes
+### Postal codes
 
-This section allows you to exclude shipping method for customers with certain ZIP codes.
-If customer shipping address ZIP Code is within any of specified ranges, that shipping method will be unavailable for that customer.
+This section allows you to exclude shipping method for customers with certain postal codes.
+If customer shipping address postal code is within any of specified ranges, that shipping method will be unavailable for that customer.
 
-For now the only supported mode for ZIP Codes is *exclude*, support for inclusion will be added in future.
+For now the only supported mode for postal codes is *exclude*, support for inclusion will be added in future.
```

Comment:
For now, the only supported mode for postal codes is *exclude*. Support for inclusion will be added in the future.